From 1c3a109df54b6092fa85a1fe2b7771e3b959655f Mon Sep 17 00:00:00 2001
From: "Krishnanjanappa, Jagadeesh"
 <jagadeesh.krishnanjanappa@caviumnetworks.com>
Date: Wed, 8 Apr 2015 18:08:14 +0530
Subject: [PATCH] dpkg: add triplet entry to fix build error for armeb
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

[PATCH] add armeb triplet entry into triplettable.

Cross-compling dpkg application for armeb fails with below error
during configure task,

(snip)
 configure:23141: checking dpkg cpu type
 configure:23148: result: armeb
 configure:23150: WARNING: armeb not found in cputable
 configure:23162: checking dpkg operating system type
 configure:23169: result: linux-gnueabi
 configure:23171: WARNING: linux-gnueabi not found in ostable
 configure:23183: checking dpkg architecture name
 configure:23189: error: cannot determine host dpkg architecture
-- CUT --

the required combination of "gnueabi-linux-armeb" was not found in
the triplettable file thereby returning dpkg_arch as
empty in configure script.

Upstream-Status: Pending

Signed-off-by: Krishnanjanappa, Jagadeesh <jagadeesh.krishnanjanappa@caviumnetworks.com>
Signed-off-by: Aníbal Limón <anibal.limon@linux.intel.com>
---
 data/tupletable | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/tupletable b/data/tupletable
index ae9f2ddb4..7436f8056 100644
--- a/data/tupletable
+++ b/data/tupletable
@@ -27,6 +27,7 @@ eabihf-musl-linux-arm		musl-linux-armhf
 base-musl-linux-<cpu>		musl-linux-<cpu>
 eabihf-gnu-linux-arm		armhf
 eabi-gnu-linux-arm		armel
+eabi-gnu-linux-armeb		armeb
 abin32-gnu-linux-mips64r6el	mipsn32r6el
 abin32-gnu-linux-mips64r6	mipsn32r6
 abin32-gnu-linux-mips64el	mipsn32el
